### PR TITLE
Update go code excerpts

### PR DESCRIPTION
**Pull request by svrnm**
I ran `npm run codeexcerpts` to apply latest changes to the go examples, I think this is one of the first times we do the update, so please take a close look

### CODE REVIEW

**Comment by dmathieu** [comment]
This seems wrong, as it removes the log example.

**Comment by svrnm** [comment]
> This seems wrong, as it removes the log example.

I was surprised about that myself... wait I might have something wrong locally.

File: content/en/docs/languages/go/getting-started.md [hunk 1]
```diff
@@ -409,30 +401,6 @@ func init() {
 		panic(err)
 	}
 }
-
-func rolldice(w http.ResponseWriter, r *http.Request) {
```
Comment (svrnm) [inline content/en/docs/languages/go/getting-started.md:413]: This has been moved to https://github.com/open-telemetry/opentelemetry-go/blob/main/example/dice/rolldice.go so we need to update the documentation accordingly

**Comment by svrnm** [comment]
turning this into a draft for now, this needs more thought than I initially anticipated

**Review by chalin — COMMENTED** [review]
Thanks @svrnm. Btw, this sort of experience goes to show that we can automate the code-snippet refresh (which is super great for minor changes), but when major structural changes are involved, we still need to carefully address the changes in the prose.

/cc @zeitlinger

**Comment by zeitlinger** [comment]
> Thanks @svrnm. Btw, this sort of experience goes to show that we can automate the code-snippet refresh (which is super great for minor changes), but when major structural changes are involved, we still need to carefully address the changes in the prose.
> 
> /cc @zeitlinger

for java, I have created a dedicated `doc-snippets` project to make this less likely

**Comment by svrnm** [comment]
> > Thanks @svrnm. Btw, this sort of experience goes to show that we can automate the code-snippet refresh (which is super great for minor changes), but when major structural changes are involved, we still need to carefully address the changes in the prose.
> > /cc @zeitlinger
> 
> for java, I have created a dedicated `doc-snippets` project to make this less likely

Can you share that project?

**Comment by zeitlinger** [comment]
> > > Thanks @svrnm. Btw, this sort of experience goes to show that we can automate the code-snippet refresh (which is super great for minor changes), but when major structural changes are involved, we still need to carefully address the changes in the prose.
> > > /cc @zeitlinger
> > 
> > 
> > for java, I have created a dedicated `doc-snippets` project to make this less likely
> 
> Can you share that project?

https://github.com/open-telemetry/opentelemetry-java-examples/pull/422

**Comment by svrnm** [comment]
Will raise a new PR because of https://github.com/open-telemetry/opentelemetry.io/pull/4679

**Pull request closed without merging**